.github/workflows/nightly-tfe-test.yml

```diff
@@ -98,6 +98,38 @@ jobs:
           fi
           exit 1
 
+  slack-notify:
+    needs: tests-summarize
+    if: always() && (needs.tests-summarize.result == 'failure')
```

Comment:
The previous job, `tests-summarize`, will fail whenever the job `tests` does not have the status "success". That means that the `slack-notify` job will run if, for example, we manually cancel the tests from running. To avoid this behavior, we could modify this line in the tests-summarize job `if [ "${{ needs.tests.result }}" = "success" ]`